src/Containers/SoldOutCardNotification/SoldOutCardNotification.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import {Card as C} from '@Containers/Card/Card'
+import styled from 'styled-components'
+import {Mixins} from '@Utils'
+
+export interface ISoldOutCardNotificationProps{
+    imgSrc:string;
+    header:React.ReactElement;
+    footer:React.ReactElement;
+    content:React.ReactElement;
+}
+
+export const SoldOutCardNotification:React.FC<ISoldOutCardNotificationProps>=({imgSrc,header,footer,content}):React.ReactElement=>
+    (
+        <Card animated widthFitContent>
+            <Container flex1='row'>
+                <Container flex1='column' flex2='space-around' flex={1}>
+                    {header}
+                    {content}
+                    {footer}
+                </Container>
+                <Container flex1='column' flex2='flex-end' flex={1}>
+                    <Img src={imgSrc} />
+                </Container>
+            </Container>
+        </Card>
+    )
+
+interface IContainerProps{
+    flex1?:string;
+    flex2?:string;
+    flex?:number;
```

Comment:
flex1, flex2, flex is not very descriptive. Would prefer flexDirection, justifyContent, and flexGrow.